unstable/wlr-workspace-unstable-v1.xml

```diff
@@ -0,0 +1,265 @@
+      </description>
+      <arg name="coordinates" type="array"/>
+    </event>
+
+    <event name="state">
+      <description summary="the state of the workspace changed">
+        This event is emitted immediately after the zwlr_workspace_handle_v1 is
+        created and each time the workspace state changes, either because of a
+        compositor action or because of a request in this protocol.
+      </description>
+      <arg name="state" type="array"/>
+    </event>
+
+    <enum name="state">
+      <description summary="types of states on the workspace">
+        The different states that a workspace can have.
+      </description>
+
+      <entry name="active" value="0" summary="the workspace is active"/>
+    </enum>
+
+    <event name="done">
+      <description summary="all information about the workspace has been sent">
+        This event is sent after all changes in the workspace state have been
+        sent.
+
+        This allows changes to one or more zwlr_workspace_handle_v1 properties
+        to be seen as atomic, even if they happen via multiple events. In
+        particular, if a workspace changes its coordinates, coordinates events
+        may be sent to multiple zwlr_workspace_handle_v1 objects as workspaces
+        are reordered. The compositor sends the done events only after all
+        coordinates events have been sent.
+      </description>
+    </event>
+
+    <event name="remove">
+      <description summary="this workspace has been destroyed">
+        This event means the zwlr_workspace_handle_v1 has been destroyed. It is
+        guaranteed there won't be any more events for this
+        zwlr_workspace_handle_v1. The zwlr_workspace_handle_v1 becomes inert so
+        any requests will be ignored except the destroy request.
+      </description>
+    </event>
+
+    <request name="destroy" type="destructor">
+      <description summary="destroy the zwlr_workspace_handle_v1 object">
+        Destroys the zwlr_workspace_handle_v1 object.
+
+        This request should be called either when the client does not want to
+        use the workspace object any more or after the remove event to finalize
+        the destruction of the object.
+      </description>
+    </request>
+
+    <request name="activate">
+      <description summary="activate the workspace">
+        Request that this workspace be activated.
+
+        There is no guarantee the workspace will be actually activated, and
+        behaviour may be compositor-dependent. For example, activating a
+        workspace may or may not deactivate all other workspaces in the same
+        group.
+      </description>
+    </request>
+
+    <request name="deactivate">
+      <description summary="activate the workspace">
+        Request that this workspace be deactivated.
+
+        There is no guarantee the workspace will be actually deactivated.
+      </description>
+    </request>
+
+    <request name="commit">
```

Comment:
Why should that be here? Can't we move commit to the manager interface, since this request seems to synchronize everything?

Reply:
Indeed, this would make more sense.

Reply:
OK, I'll make this change.